pkg/backend/display/progress.go

```diff
@@ -709,8 +716,8 @@ func (display *ProgressDisplay) processEndSteps() {
 		}
 	}
 
-	// If we get stack outputs, display them at the end.
-	if display.stackUrn != "" {
+	// If we get stack outputs and there weren't any errors, display them at the end.
+	if display.stackUrn != "" && !sawError {
```

Comment:
it's worthwhile in cases like this to explain the 'why' not the what.  i.e. why would we not want to show the error.

Reply:
will do.